vitrivr-engine-index/src/main/kotlin/org/vitrivr/engine/index/decode/VideoDecoder.kt

```diff
@@ -161,7 +161,7 @@ class VideoDecoder : DecoderFactory {
             }
 
             /* Create and emit audio content. */
-            val audio = this.context.contentFactory.newAudioContent(frame.audioChannels.toShort(), frame.sampleRate, samples)
+            val audio = this.context.contentFactory.newAudioContent(frame.audioChannels.toShort(), frame.sampleRate * 2, samples)
```

Comment:
I guess there should be a hint why audio is hardcoded sampled twice the video rate?

Reply:
So, I don't really know why, but i determined this through trial and error :) It probably is due to the frame using a different unit of sample (bytes vs shorts?) when it comes to sample rate.